plugins/lua/stockpiles.lua

```diff
@@ -542,32 +574,51 @@ function StockpilesOverlay:onRenderFrame()
     local sp = dfhack.gui.getSelectedStockpile(true)
     if sp and self.cur_stockpile ~= sp then
         local config = logistics.logistics_getStockpileConfigs(sp.stockpile_number)[1]
+        print("onRenderFrame - logistics_GET StockpileConfigs()")
+        print("melt=",config.melt,"trade=",config.trade,"dump=",config.dump,"forbid=",config.forbid,"train=",config.train)
         self.subviews.melt:setOption(config.melt == 1)
         self.subviews.trade:setOption(config.trade == 1)
         self.subviews.dump:setOption(config.dump == 1)
+        self.subviews.forbid:setOption(config.forbid+1)
         self.subviews.train:setOption(config.train == 1)
         self.cur_stockpile = sp
     end
 end
 
 function StockpilesOverlay:toggleLogisticsFeature(feature)
+    print("entering toggleLogisticsFeature - feature = ", feature)
+    --print("option value - self.subviews.forbid:getOptionValue() = ", self.subviews.forbid:getOptionValue())
+    --print("option value - self.subviews[feature]:getOptionValue() = ", self.subviews[feature]:getOptionValue())
+    --print("option index - self.subviews[feature].option_idx", self.subviews.forbid.option_idx)
```

Comment:
(and below) leftover debug prints?

Reply:
These are cleaned up along with some other improvements to readability that were intended to be in the first PR.

I did intentionally leave a couple new debug lines in logistics.cpp that seemed fitting and output there requires being enabled with `debugfilter` (there's no `debugfilter` functionality pattern already in place for lua, right?)